### PR TITLE
Rprout slate doc dev

### PR DESCRIPTION
**Pull request by proutrc**
Sorry, this should be labeled as WIP. Creating this as a "Draft Pull Request."

But, this is for the in progress Slate docs. Slate documentation begins in the high level space "Services and Applications"->"Slate".

### CODE REVIEW

**Comment by proutrc** [comment]
@kincl Please feel free to take a look at this draft. Any feedback is welcome :)

File: services_and_applications/slate/building_your_own_application/prerequisites.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+The OC tool provides CLI access to the OpenShift cluster. It needs to be installed on your machine.
+
+It is a single binary that can be downloaded from a number of places (the choice is yours):
+
+- `Homebrew <https://brew.sh/>`_ on MacOS (need Homebrew setup first): 
+
+ .. code-block:: bash
+
+     brew install openshift-cli 
+
```
Comment (kincl) [inline services_and_applications/slate/building_your_own_application/prerequisites.rst:26]: I am not sure that the homebrew package is always kept up to date so if we have it in the list I think it should be farther down and have a note

```suggestion
- `Homebrew <https://brew.sh/>`_ on MacOS (need Homebrew setup first): 

 .. note::

     The Homebrew package is not always kept up to date with the latest version of OpenShift so some client features may not be available

 .. code-block:: bash

     brew install openshift-cli 

```

File: services_and_applications/slate/building_your_own_application/prerequisites.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Follow :ref:`slate_getting_started` if you do not already have a project/namespace established on Onyx or Marble.
+
+Install the OC tool
+-------------------
+
+The OC tool provides CLI access to the OpenShift cluster. It needs to be installed on your machine.
+
+It is a single binary that can be downloaded from a number of places (the choice is yours):
+
```
Comment (kincl) [inline services_and_applications/slate/building_your_own_application/prerequisites.rst:20]:
```suggestion

* Direct from the cluster (preferred):

  * Marble `Command Line Tools <https://console-openshift-console.apps.marble.ccs.ornl.gov/command-line-tools>`_

  * Onyx `Command Line Tools <https://console-openshift-console.apps.onyx.ccs.ornl.gov/command-line-tools>`_

```

File: services_and_applications/slate/overview.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+*Slate* is the high-level service name referring to two user
+facing OpenShift clusters.
+
+| OpenShift is a container orchestration system built on top of Kubernetes and supported by Red Hat. 
+  Kubernetes provides a rich API and workload primitives that allows users to manage the application 
+  deployments of long running services such as web servers and databases.
+
+| OpenShift is a Kubernetes Certified Platform. This means that all of the functionality that is
+  in Kubernetes is also implemented in OpenShift. OpenShift has extra functionality built on 
+  top though, such as the container builder and integrated registry services.
```
Comment (kincl) [inline services_and_applications/slate/overview.rst:14]: I say we move this to under the "What is OpenShift" header at the bottom

File: services_and_applications/slate/overview.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
+*****************************
+Overview
+*****************************
+
+*Slate* is the high-level service name referring to two user
+facing OpenShift clusters.
```
Comment (kincl) [inline services_and_applications/slate/overview.rst:6]:
```suggestion
*Slate* provides container orchestration services that support a user project's existing Summit and Alpine allocations. The Slate service today consists two user facing OpenShift clusters which each provide capabilities unique to the security enclave that they are in.
```

Comment (kincl) [reply services_and_applications/slate/overview.rst:6]: I don't love the wording but I think we should talk more about what Slate is overall

File: services_and_applications/slate/building_your_own_application/prerequisites.rst (hunk 1 above)
Comment (proutrc) [reply services_and_applications/slate/building_your_own_application/prerequisites.rst:32]: I did not know you could get directly form the cluster, that is awesome to know. I will probably remove the brew install option, if that is not kept up to date.

File: services_and_applications/slate/overview.rst (hunk 3 above)
Comment (proutrc) [reply services_and_applications/slate/overview.rst:13]: sound good. will do. Thanks!

**Comment by jack-morrison** [comment]
It will be great to have Slate docs as part of this repo. Do we have a graphic that draws the distinction between OLCF's HPC resources from Slate resources? I can see words like "node" being confusing for someone unfamiliar with the dedicated hardware in Slate clusters, especially since there exists some capability to integrate with the HPC systems.

**Pull request merged**